onnxruntime/core/providers/cuda/reduction/reduction_functions.cc

```diff
@@ -60,11 +60,11 @@ optional<std::pair<int64_t, int64_t>> GetMinAndMaxContiguousAxes(
     }
 
     // otherwise, not contiguous
-    return nullopt;
+    return std::nullopt;
```

Comment:
>std::nullopt; [](http://example.com/codeflow?start=11&length=13)

using std::nullopt with onnxruntime::optional here, can we use std or onnxruntime consistently?

Reply:
Done.